components/Reviews.tsx

```diff
@@ -0,0 +1,130 @@
+import { Button, Card, Select, Text, User } from '@geist-ui/react';
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { STORY_REVIEW_STATUS, STORY_REVIEW_TYPES } from '../constants';
+import { getPeople } from '../redux/selectors/projects.selectors';
+import { Review } from '../redux/types';
+
+interface ReviewParams {
+  reviewsData: Review[];
+  currentState: any;
+  storyId: number;
+  updateStory: (state: any) => void;
+}
+
+const ReviewCard = styled(Card)`
+  width: max-content;
+`;
+
+const ReviewSelect = styled(Select)`
+  margin-left: 8px;
+`;
+
+const AddReviewContainer = styled.div`
+  display: flex;
+`;
+
+const Reviews = ({
+  reviewsData,
+  storyId,
+  currentState,
+  updateStory,
+}: ReviewParams): JSX.Element => {
+  const [addingReview, setAddingReview] = useState(false);
+  const people = useSelector(getPeople);
+
+  const addReview = (reviewType: number) => {
+    const { reviews } = currentState;
+    const newReview = {
+      id: Date.now(), // this is a temporal ID to identify review until save
```

Comment:
I've just been using `pending` for this on stories, since we can only have one new/unsaved at a time, it's a magic number but makes it easy to remove later.

Reply:
I saw that but here I intended to add several at the same time, and then do multiple requests. If this will be a problem I can surely switch to one at a time.

Reply:
Reasoning is solid! Carry on!